src/utilities/findBreakingChanges.ts

```diff
@@ -131,6 +157,13 @@ function findDirectiveChanges(
     });
   }
 
+  for (const newDirective of directivesDiff.added) {
+    schemaChanges.push({
+      type: SafeChangeType.DIRECTIVE_ADDED,
```

Comment:
Can we really say this safe?

Reply:
Yes, a directive that gets added isn't neccasrily used so why would that be unsafe. Adding a directive to a field cvould be considered breaking but this never